lib/database/postgresql.go

```diff
@@ -0,0 +1,83 @@
+package database
+
+import (
+	"context"
+	"fmt"
+
+	_ "github.com/jackc/pgx" // PostgrerSQL driver
+	"github.com/jackc/pgx/v4"
+	"github.com/sdslabs/gasper/configs"
+	"github.com/sdslabs/gasper/types"
+)
+
+var (
+	postgresqlHost         = `%`
+	postgresqlRootUser     = configs.ServiceConfig.Kaen.PostgreSQL.Env["POSTGRES_USER"]
+	postgresqlPassword     = configs.ServiceConfig.Kaen.PostgreSQL.Env["POSTGRES_PASSWORD"]
+	postgresqlDatabaseName = configs.ServiceConfig.Kaen.PostgreSQL.Env["POSTGRES_DB"]
+	postgresqlPort         = configs.ServiceConfig.Kaen.PostgreSQL.ContainerPort
+)
+
+// CreatePostgresqlDB creates a postgre database
+func CreatePostgresqlDB(db types.Database) error {
+	ctx := context.Background()
+	connection := fmt.Sprintf("postgres://%v:%v@localhost:%d/%v", postgresqlRootUser, postgresqlPassword, postgresqlPort, postgresqlRootUser)
+	conn, err := pgx.Connect(ctx, connection)
+	if err != nil {
+		return fmt.Errorf("Error while creating the database : %s", err)
+	}
+	defer conn.Close(ctx)
+
+	if _, err = conn.Exec(ctx, "CREATE DATABASE "+db.GetName()); err != nil {
+		return fmt.Errorf("Error while creating the database : Database Already Exists")
+	}
+
+	query := fmt.Sprintf("CREATE USER %s WITH PASSWORD '%s'", db.GetUser(), db.GetPassword())
+	if _, err = conn.Exec(ctx, query); err != nil {
+		if err = refrershPostgreUser(db, conn); err != nil {
+			return fmt.Errorf("Error while creating the database : %s", err)
+		}
+	}
+
+	query = fmt.Sprintf("GRANT ALL PRIVILEGES ON DATABASE \"%s\" TO %s", db.GetName(), db.GetUser())
```

Comment:
Why use a different string literal than in here? (bottom):
https://github.com/sdslabs/gasper/pull/197/files#diff-fde21d376642378ecd0deb39fc906081R35
> Using apostrophe instead of double quotes or vice-versa